src/Nest/Document/Multiple/Bulk/BulkOperation/BulkUpdate.cs

```diff
@@ -47,6 +47,12 @@ public interface IBulkUpdateOperation<TDocument, TPartialDocument> : IBulkOperat
 		long? IfSequenceNumber { get; set; }
 
 		long? IfPrimaryTerm { get; set; }
+
+		/// <summary>
+		/// True or false to return the _source field or not, or a list of fields to return.
+		/// </summary>
+		[DataMember(Name = "_source")]
+		bool? Source { get; set; }

```

Comment:
I think this should be a `Union<bool, ISourceFilter>`, based on what `UpdateRequest` supports for `_source`:

https://github.com/elastic/elasticsearch-net/blob/c3301149632df4611bb09da7eb0526db6f3d8ae0/src/Nest/Document/Single/Update/UpdateRequest.cs#L31-L32